generator/templates/default/build/manifest.json

```diff
@@ -3,7 +3,7 @@
   "namespace": "<%= bundleNamespace %>-",
   "version": 1,
   "styles": [
-    "index.css"
+    "bundle.css"
```

Comment:
Vue CLI build with `lib` target produces `common` and `umd` build. Since it isn't configurable to produce only one build, `cleanup.js` was added to build process. It removes `common` build, and renames files. 
I can't quite remember why was `bundle.css` renamed to `index.css` (on line 25). Considering this change in manifest, that part should be removed, otherwise bundle registration/update would fail due to file names mismatch.